README.md

```diff
@@ -1,273 +1,116 @@
+and debug your apps in development and in production.
 
-It is recommend to install the process manager as a system service, see below,
-but if you are just trying the manager out to try it, it can be run directly
-from the command line.
+For more details, see http://strong-pm.io.
 
-Run process manager:
+## Features
 
-    sl-pm
+- Build, package, and deploy your Node application to a local or remote system.
+- Manage processes and cluster (use all the cores on your server), adjust sizes, built-in load balancer
+- Automatic restarts keep applications running forever
+- Zero-downtime redeploy/upgrade
+- View performance metrics on your application
+- View CPU profiles and heap snapshots to optimize performance and diagnose memory leaks
+- Use graphical tool StrongLoop Arc, **slc arc**, or command-line tool, **slc**
 
-Install [strong-build](http://github.com/strongloop/strong-build) and
-[strong-deploy](http://github.com/strongloop/strong-deploy):
+*** XXX We should put the comparison table right here methinks ***
 
-    npm install -g strong-build
-    npm install -g strong-deploy
+### Build & Deploy
 
-Clone, build, and deploy an app, the loopback example app in this example, but
-any node application can be managed:
+- Automated build and multi-host deploy.
+- Zero-downtime application restarts and upgrades.
+- Install dependencies, run custom build steps, and prune development dependencies without affecting your source tree.
+- Deploy without dependence on outside servers (like NPM)
+- Create an npm package or commit the build onto a Git deploy branch.
 
-    git clone git@github.com:strongloop/loopback-example-app.git
-    cd loopback-example-app
-    sl-build
-    sl-deploy
+### Profile
 
-## Life-cycle
+- Heap snapshots and CPU profiles.
+- Profile local or remote applications with Arc.
+- Profile local applications with slc.
+- Use the _watchdog_ to start CPU profiling when the Node event loop stalls. ***XXX we should really remove this feature and only make it available through Arc... different issue though... maybe just remove for now.***
 
-When applications are deployed to the manager, it first prepares them. The
-prepare commands used are:
+### Process Manager
 
-- `npm rebuild`
-- `npm install --production`
+- Local and remote process management for Node & Express apps.
+- Log aggregation and management.
+- Change cluster size, view clustering info remotely.
+- Manage Nginx load-balancer for multi-host deployments.
 
-Since `npm install` is called, the preparation may be customized using npm
-install and pre-install scripts, if necessary.
+### Metrics
 
-After preparation, the application is run.
+- View performance metrics in Arc
+- Use your own StatsD console if you prefer.
 
-## Installing as a Service
+***XXX I think above should enumerate the 3rdparty integrations both here and on
+strong-pm.io, I'll resync with strong-pm.io after Rand has updated it, see
+original Feature list - agree --Issac***
```

Comment:
@crandmck When you have text for this, please ping me, and I'll sync back up against the web site for the feature list.

Or you can edit this file in place, if you prefer.